src/tokenize_sentence/mod.rs

```diff
@@ -0,0 +1,83 @@
+// Copyright 2019 vtext developers
+//
+// Licensed under the Apache License, Version 2.0,
+// <http://apache.org/licenses/LICENSE-2.0>. This file may not be copied,
+// modified, or distributed except according to those terms.
+
+/*!
+# Sentence tokenization module
+
+For instance let's tokenize the following text using the Unicode segmentation
+```rust
+# let s = "Here is one. Here is another! This trailing text is one more";
```

Comment:
Maybe let's keep this one uncommented to make it more clear what we are tokenizing?